raster-foundry/raster-foundry
# Add shape filter to scenes in api and hook up frontend

## Overview

This PR:
* Adds backend support to enable scene filter based on an existing shape in the shapes table for Raster Foundry repo
* Updates frontend shapes filter to enable it for Raster Foundry data repo
* Fixes a minor frontend issue to let shape filter persist when other filters are also applied

### Checklist

- ~Styleguide updated, if necessary~
- ~Symlinks from new migrations present or corrected for any new migrations~

### Notes

Spec update is in https://github.com/raster-foundry/raster-foundry-api-spec/pull/7


## Testing Instructions

 * Go to a project and browse scenes
 * See if the shape filter is in the filter pane under raster foundry repo
 * If yes, apply shape filter along with other filters and see if it behaves as you would expect
 * Switch to other data repo, e.g. Planet Labs, to see if shape filter and other filters work as you would expect

Closes #2893
